### PR TITLE
Fix issues with loading JSON and add basic documentation.

### PR DESCRIPTION
**Pull request by DiegoMagdaleno**
- Fixes an issue with loading the JSON file, the code is replaced by `json.load` instead of `json.loads`.

- Adds basic tutorial on how to run the bot.

### CODE REVIEW

**Comment by Kab1r** [comment]
So here is the thing: I have this bot hosted on Heroku, so I am actually storing the contents of the JSON file in the environment variable. When`json.loads` is used, we are parsing the variable, not the file that it points to.

I do appreciate the documentation.

File: run_bot.py [hunk 1]
```diff
@@ -132,15 +132,11 @@ async def delete_message(likelihood, message):
 # Creates Vission Client
 # Gets token and credentials from Env Var
 
+service_account_info = json.load(
+    open(os.environ.get("GOOGLE_APPLICATION_CREDENTIALS")))
 VISIONARY = vision.ImageAnnotatorClient(
-    credentials=service_account.Credentials.from_service_account_info(
-        json.loads(
```
Comment (Kab1r) [inline run_bot.py]: Revert to loads?

**Review by Kab1r — CHANGES_REQUESTED** [review]

**Comment by DiegoMagdaleno** [comment]
@KabirKwatra Now it will detect if its a file and use load if not if it will use loads, that way nothing should break and allows people to export their env variable as whatever they consider better.

**Review by Kab1r — APPROVED** [review]

**Comment by Kab1r** [comment]
Merged in #3

**Pull request closed without merging**